packages/providers/test/provider.test.ts

```diff
@@ -56,7 +56,8 @@ const getCustomFetch =
     }
   ) => {
     const graphqlRequest = JSON.parse(options.body);
-    const { operationName } = graphqlRequest;
+    const { query } = graphqlRequest;
+    const operationName = query.split('{')[0].split(' ')[1].split('(')[0]; // e.g. query getMessageProof(...) {...} -> getMessageProof
```

Comment:
This must be improved.

Reply:
[acfa244](https://github.com/FuelLabs/fuels-ts/pull/1597/commits/acfa2449d11bea7699da1de189856ba9498ca912)

Reply:
I think you didn't get it—_we can't outsource this problem_.

This info should be easily accessible by external users, as it was before.

Reply:
I'm still not getting it. Are you referring to the `graphqlRequest` variable that doesn't have `operationName` directly on it or to the way I'm getting the `operationName` variable?

Reply:
You shouldn't modify existing tests to fit your [new] implementation if it was supposed to be a drop-in replacement.

You are outsourcing the burden of figuring out how to extract `operationName` to external users:

```ts
const { query } = graphqlRequest;
const operationName = query.split('{')[0].split(' ')[1].split('(')[0]; // e.g. query getMessageProof(...) {...} -> getMessageProof
```

It should be just like it was before:

```ts
const { operationName } = graphqlRequest;
```

Reply:
Aha, I understand. The `graphql-request` library was adding an `operationName` property to the request body alongside the `query` and `variables` properties. The `query` and `variables` are the important ones, however. After I replaced it, tests that were using this `getCustomFetch` helper started to fail because this `operationName` property was missing from the body. IMO this property is unnecessary and there's no need for us to actually add it to each one of our requests as it doesn't add any production value and we can make this helper happy with the implementation I presented in the commit above. 

After this additional context, if you still feel that the `operationName` field should be there - it's easy to reintroduce it.